contracts/seed/Seed.sol

```diff
@@ -127,112 +131,124 @@ contract Seed {
       * @param _tokens                Array containing two params:
                                         - The address of the seed token being distributed.
       *                                 - The address of the funding token being exchanged for seed token.
-      * @param _softAndHardCap         Array containing two params:
+      * @param _softHardReq           Array containing two params:
```

Comment:
can we not use abbreviations?  I am a native english speaker and still no idea with "Req" is here

Reply:
why have we decided to move from `_softAndHardCap`?

Reply:
Because, As Doug wants to send the required seed amount from deploy seed, and to do that we need one more variable. If we add one more, it gives ```stack too deep```, so what I did was, we pass an array with three variables,
Index 0 - ```softCap```
Index 1 - ```hardCap```
Index 2 - it can be anything, as in the ```deploySeed```, I store the value of required seed amount at this index, which will passed to ```Seed.initialise``` function. So that we don't use ```seedToken.balanceOf()``` as Doug mentioned.

So to cover this three I renamed it, should we go with ```_softAndHardCap``` only?

Reply:
Added to Issue #160 , This PR is getting very big. Closing this request.